hoomd_benchmarks/microbenchmark_custom_force.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2021-2022 The Regents of the University of Michigan
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Custom Force benchmark."""
+
+import hoomd
+import numpy as np
+from . import common
+from .configuration.hard_sphere import make_hard_sphere_configuration
+
+try:
+    import cupy as cp
+    CUPY_IMPORTED = True
+except ImportError:
+    CUPY_IMPORTED = False
+
+
+class EmptyForce(hoomd.md.force.Custom):
+    """Custom force which calls an empty set_forces method."""
+
+    def __init__(self):
+        super().__init__()
+
+    def set_forces(self, timestep):
+        """Set the forces."""
+        pass
+
+
+class ConstantForce(hoomd.md.force.Custom):
```

Comment:
@joaander did you want the test to test a force like gravity or something like `arrays.force[:] = np.array([0, 0, 1]`?

I can see the usefulness of both but I was just wondering.

Reply:
I don't have a strong preference. I was originally thinking of the constant force which would be measurement of how much slowdown using `Custom` as a replacement for the new removed `Constant` force.